### PR TITLE
Documentation Update: Add from and to usage examples to migration script samples

### PR DESCRIPTION
**Pull request by ccjx**
This updates the documentation to add a from and to usage examples to migration script samples on the Migrations page under the General SQL Scripts section. I believe that this will help new users of EF code-first as there are no examples of its usage in the current EF Core documentation.

### CODE REVIEW

File: entity-framework/core/managing-schemas/migrations/index.md [hunk 1]
```diff
@@ -209,16 +209,44 @@ When debugging your migrations or deploying them to a production database, it's
...
 
+#### Basic Usage
 ``` powershell
 Script-Migration
 ```
 
+#### With From (to implied)
+This will generate a SQL script from this migration to the latest migration.
+```powershell
+Script-Migration -From 20190725054716_Add_new_tables
```
Comment (bricelam) [inline entity-framework/core/managing-schemas/migrations/index.md]: Let's be consistent and not show `-From` and `-To` on PowerShell

Comment (ccjx) [reply entity-framework/core/managing-schemas/migrations/index.md]: Thanks @bricelam, I've updated the file.

**Comment by ccjx** [comment]
@bricelam do I need to do anything else to get this merged?

**Comment by ajcvickers** [comment]
Thanks!

**Pull request merged**